packages/richtext-lexical/src/features/migrations/lexicalPluginToLexical/nodes/unknownConvertedNode/index.tsx

```diff
@@ -33,7 +33,7 @@ export class UnknownConvertedNode extends DecoratorNode<JSX.Element> {
   }
 
   static override clone(node: UnknownConvertedNode): UnknownConvertedNode {
-    return new UnknownConvertedNode({
+    return new this({
```

Comment:
I don't think these changes in clone are necessary. It's already handled by `$applyNodeReplacement`. The Lexical codebase doesn't use this pattern.

Reply:
I mainly did this for consistency within our codebase. We use `this` pattern ( ;) ) for our ServerBlockNode and BlockNode, so that we don't have to override `clone` in `BlockNode` when we extend from `ServerBlockNode`.

We don't use this `ServerUnknownConvertedNode` here, but consistency I just used `this` all across our codebase